frontend/src/pages/Favourite/Favourite.tsx

```diff
@@ -1,46 +1,69 @@
 import {useEffect, useState} from "react";
-import showMyFavourites, {deleteFromFavourites, getDrink} from "../../apiServices/service";
-import {Cocktail} from "../../components/Model";
+import {deleteFromFavourites, getCustomDrink, getDrink, showMyFavourites} from "../../apiServices/service";
+import {CocktailModel, CustomDrinkModel} from "../../components/Model";
 import {useNavigate} from "react-router-dom";
 import "./Favourite.css"
 
 
+
+
 export default function Favourite() {
 
-    const [favourites, setFavourites] = useState<Cocktail[]>([])
+    const [favourites, setFavourites] = useState<CocktailModel[]>([])
+    const [customFavourites, setCustomFavourites] = useState<CustomDrinkModel[]>([])
 
 
     const nav = useNavigate()
 
 
     useEffect(() => {
         getFavourites()
-
-    }, [])
+// eslint-disable-next-line react-hooks/exhaustive-deps
+    },[])
```

Comment:
Here your missing something in the dependency array, we should fix it, not disable it